src/main.js

```diff
@@ -25,13 +25,14 @@ async function main() {
 
   var pkg = await readPkg(pkgFile)
 
+  pkg.name = (pkg.name || 'unknown').split('/').pop()
```

Comment:
I don't think `readPkg` works if there's no name, but in any event if there's no name we should just exit early with an error rather than default a name.